src/components/Jobs/Jobs.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
+import { FormattedMessage, injectIntl, intlShape } from 'react-intl';
```

Comment:
In Epam we are using such code style. Feel free to join to our side)
```suggestion
import { 
  FormattedMessage, 
  injectIntl, 
  intlShape,
} from 'react-intl';
```

Reply:
Do y'all have a strict rule about when to make the import multi-line vs. single-line? Is it just any list of imports greater than one is multi-line? (I'll happily follow your conventions on that, I haven't been consistent about it.)

Reply:
Yeah you right. It Is just any list of imports greater than one is multi-line.